lexmapr/pipeline.py

```diff
@@ -195,7 +196,7 @@ def run(args):
     remainingAllTokensSet = []
     remainingTokenSet = []
     prioritizedRetainedSet=[]
-    samplesDict = {}
+    samplesDict = collections.OrderedDict()
```

Comment:
This guarantees the order of the samples so that the output rows are always printed in the same order.